lib/PuppeteerSharp.DevicesFetcher/PuppeteerSharp.DevicesFetcher.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
These projects are for internal use; we can bump them to 2.1 instead of making them multi-target.